app/src/androidTest/java/com/github/steroidteam/todolist/MapsActivityTest.java

```diff
@@ -56,6 +67,36 @@ public void markerIsCorrectlyPlacedAtDefaultLocationOrUserLocation() {
         }
     }
 
+    @Test
+    public void searchViewWorksCorrectlyWithCorrectLocation() {
+        String TEST_LOCATION = "Lausanne";
+        onView(withId(R.id.sv_location))
+                .perform(click(), typeText(TEST_LOCATION), pressKey(KeyEvent.KEYCODE_ENTER));
+        waitFor(4000);
```

Comment:
Why do you need to wait ?

Reply:
Because there is an animation that moves to the searched place, so I wait until the animation have finished to test the marker

Reply:
Perhaps we should deactivate animation during tests...